nucliadb/nucliadb/search/search/merge.py

```diff
@@ -233,12 +234,22 @@ async def merge_vectors_results(
         start, end = position.split("-")
         start_int = int(start)
         end_int = int(end)
-        index_int = int(index)
+        try:
+            index_int = int(index)
```

Comment:
Why `index` can be non numeric? Can you add a comment for this?

Reply:
In vectoret index is a STR on processing one is the position (so its an int)